core/backend/src/IModelDb.ts

```diff
@@ -164,12 +172,31 @@ export abstract class IModelDb extends IModel {
     const eventSinkId = `${this._fileKey}-${(IModelDb._nextEventSinkId++).toString()}`;
     this._eventSink = new EventSink(eventSinkId);
     this.nativeDb.setEventSink(this._eventSink);
+    IModelDb._openDbs.set(this._fileKey, this);
   }
+
+  /** Close this IModel, if it is currently open.
+   * @throws IModelError if the iModel is not open

```

Comment:
nitpick but does it actually throw? Looks like it just returns.

Reply:
fixed.